odoo/tools/misc.py

```diff
@@ -1221,7 +1227,7 @@ def format_date(env, value, lang_code=False, date_format=False):
         else:
             value = odoo.fields.Datetime.from_string(value)
 
-    lang = env['res.lang']._lang_get(lang_code or env.context.get('lang') or 'en_US')
+    lang = get_lang(env, lang_code)
```

Comment:
But then I would have to do it at lines 1250, 1281, 1311 and 1329, wouldn't I? This is the kind of things that made one method work but not the others because a fix is rarely done everywhere (like it should) but only at one place.